utilities/blob_db/blob_db.h

```diff
@@ -199,6 +203,9 @@ class BlobDB : public StackableDB {
     return NewIterator(options);
   }
 
+  using rocksdb::StackableDB::Close;
+  virtual Status Close() override = 0;
```

Comment:
@anand1976 I think we have a problem now with Close() still being virtual and people override that instead of CloseImpl().  This will make a confusing experience with some DB overriding Close() and others CloseImpl(). Should Close() be made non-virtual? What was the final decision?

Reply:
Yeah I agree it should be non-virtual.. if people override `Close()` they can't set `closed_ = true` and then `DBImpl::CloseHelper` will always be called in destruction.

Reply:
Oh ignore my comment. This `StackableDB` class doesn't subclass `DBImpl` so my comment was wrong.

Reply:
Discussed with @anand1976 and @ajkr. I'm going to implement CloseImpl() not Close(). I'll split the change to another PR for review.

Reply:
There is no DB::CloseImpl(). The Close()/CloseImpl() pattern was only for the Logger class. Although after talking with @yiwu-arbug and @ajkr it makes sense to have the same pattern in DB as well.

Reply:
I'll go ahead with my current PR and wait till changes on DBImpl side to change to use CloseImpl().